### PR TITLE
session-desktop: init at 1.0.8

### LINKED ISSUE #81927: Package request: Session Messenger
**Project description**
"Session is an end-to-end encrypted messenger that removes sensitive metadata collection, and is designed for people who want privacy and freedom from any forms of surveillance."
Session is a fork of [Signal](https://www.signal.org/) designed to be used on top of the [Loki Network](https://loki.network/) for onion routing and storage. 

The existing `signal-desktop` package can probably be adapted for this.
**Metadata**

* homepage URL: https://getsession.org/
* source URL: https://github.com/loki-project/session-desktop
* license: GPL3 ...
* platforms: linux, electron ...

### PR DESCRIPTION
**Pull request by mehlon**
###### Motivation for this change
Fixes #81927

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by Emantor — APPROVED** [review]
Builds and starts correctly on NixOS unstable.

**Comment by LEXUGE** [comment]
Any update here?

**Comment by billksun** [comment]
What needs to be done for this to be merged?

**Comment by mehlon** [comment]
This is functional and ready to integrate, although I am personally not satisfied with using a pre-built AppImage.
I would prefer to build it from source, but I have not felt the need for this software, so I haven't worked on it.

**Comment by ryantm** [comment]
@mehlon Are you willing to add yourself as the maintainer of this?

File: pkgs/applications/networking/instant-messengers/session-desktop/default.nix [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    mv $out/bin/${name} $out/bin/${pname}
+    install -m 444 -D ${appimageContents}/${pname}.desktop $out/share/applications/session.desktop
+    install -m 444 -D ${appimageContents}/${pname}.png $out/share/icons/hicolor/512x512/apps/session.png
+    substituteInPlace $out/share/applications/session.desktop --replace 'Exec=AppRun' 'Exec=${pname}'
+  '';
+
+  meta = with lib; {
+    description = "Desktop client for Session Messenger";
+    homepage = https://getsession.org/;
+    license = licenses.gpl3;
```
Comment (SuperSandro2000) [inline pkgs/applications/networking/instant-messengers/session-desktop/default.nix:33]: plus or Only?

File: pkgs/applications/networking/instant-messengers/session-desktop/default.nix [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    #maintainers = with maintainers; [ ];
```
Comment (SuperSandro2000) [inline pkgs/applications/networking/instant-messengers/session-desktop/default.nix:34]: Please add yourself as a maintainers.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by mehlon** [comment]
Sorry, I won't be maintaining this package.

**Comment by SuperSandro2000** [comment]
> Sorry, I won't be maintaining this package.

Adding packages without any maintainer basically marks them to be eventually broken. Maybe we shouldn't add the package if no one is planning to maintain it.

**Comment by ryantm** [comment]
Yes, we can't accept this without a maintainer. If someone wants to pick that up, we can reopen.

**Pull request closed without merging**